vtex-sites/base.store
# feat: Always enable refresh endpoint

## What's the purpose of this pull request?
Gatsby has this feature called "refresh endpoint". This is necessary for the preview server to work. For more info, [check their docs](https://www.gatsbyjs.com/docs/how-to/local-development/running-a-gatsby-preview-server/). 

I'm adding this as a default because it seems to me this is a harmless feature to have running as default and will allow developers working with headless CMS to have a better DX

## How it works? 
When the developer is working with a CMS and wants to preview/upgrade any data, the developer may run:
```
curl -X POST http://localhost:8000/__refresh
```

This will hit the refresh endpoint and update the data, like in a live preview version and the dev won't have to restart the Gatsby process

## How to test it?
Run with a normal `yarn develop` and apply the aforementioned curl. This should refresh the data on Gatsby data layer

## References
[Gatsby Preview Server](https://www.gatsbyjs.com/docs/how-to/local-development/running-a-gatsby-preview-server/)
